projects/core/src/cart/facade/cart.service.ts

```diff
@@ -194,12 +174,21 @@ export class CartService {
 
   getEntry(productCode: string): Observable<OrderEntry> {
     return this.store.pipe(
-      select(fromSelector.getEntrySelectorFactory(productCode))
+      select(fromSelector.getCartEntrySelectorFactory(productCode))
     );
   }
 
   isCreated(cart: Cart): boolean {
-    return cart && !!Object.keys(cart).length;
+    return (
+      cart && !!Object.keys(cart).length && typeof cart.guid !== 'undefined'
+    );
+  }
+
+  isLoaded(cart: Cart): boolean {
```

Comment:
Can we move those 3 helper methods (isCreated, isLoaded, isEmpty) to separate service, i.e. CartUtilService? It will leave a space for analogical methods in the CartService facade for future.